frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -701,7 +754,7 @@ class DatasetImportView extends React.PureComponent<Props, State> {
           bordered={false}
           title={
             <h3>
-              {titleString} Dataset: {this.props.datasetId.name}
+              {titleString} Dataset: {this.props.datasetId.name} (hasUnsavedChanges:{" "}
```

Comment:
I think this change needs a little more work 🙈 .
Maybe you can pet it? 🐶 

I suggest to display an icon here in case there are unsaved changes so the user sees that she/he still needs to save before leaving. 
A smart guess from my side: I think that was already your idea but you to distracted while working on this and then missed to finish it 🐱

Reply:
yeah, sorry, this was only meant as a debugging help :sweat_smile: I'll remove it :)